### PR TITLE
ci: pin to nodejs workflows next and releases branches

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

- Pin `next` and `**-release` branches to `if-nodejs-release` workflows.
- Update `if-nodejs-version-bump` workflow including `next` branch.

`next` branch at the moment is only used in `react-component` repo.

### CODE REVIEW

**Review by derberg — COMMENTED** [review]
Just revert changes you have done to ` .github/workflows/if-nodejs-version-bump.yml` and only modify line `ref: master` to below

```
          # target branch of release. More info https://docs.github.com/en/rest/reference/repos#releases
          ref: ${{ github.event.release.target_commitish }} #in case release is created from release branch then we need to checkout from given branch
```

File: .github/workflows/if-nodejs-release.yml [hunk 1]
```diff
@@ -7,6 +7,8 @@ on:
   push:
     branches:
       - master
+      - next
```
Comment (derberg) [inline .github/workflows/if-nodejs-release.yml:10]:
```suggestion
      #below lines are not enough to have release supported for these branches
      #make sure configuration of `semantic-release` package mentiones these branches
      - next
```

**Review by derberg — APPROVED** [review]
lgtm 🚀

**Comment by magicmatatjahu** [comment]
@derberg Thanks! 🚀

**Pull request merged**